src/components/organisms/Feedbacks/FeedbackCard.jsx

```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import Card, { variants as cardVariants } from '../../atoms/Card';
+
+const Photo = styled.div`
+  flex: none;
+
+  width: 4em;
+  height: 4em;
+
+  background-color: ${({ theme }) => theme.colors.secondary};
+
+  border-radius: 50%;
+`;
+
+const Image = styled.img`
+  display: block;
+  width: 100%;
+  height: auto;
+
+  border-radius: 1em;
+`;
+
+const Person = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  margin: -0.5em;
+
+  & > * {
+    margin: 0.5em;
+  }
+`;
+
+const PersonDescription = styled.div``;
+
+const Title = styled.h3`
+  margin: 0 0 0.5em;
+
+  color: ${({ theme }) => theme.colors.text};
+  font-weight: bold;
+`;
+
+const SubTitle = styled.p`
+  margin: 0;
+
+  color: ${({ theme }) => theme.colors.paleText};
+`;
+
+const FeedbackCardWrapper = styled(Card)`
+  break-inside: avoid;
+
+  max-width: 100%;
+
+  margin-bottom: 2em;
```

Comment:
It shouldn't be here. These properties is about layouting. Element itself shouldn't know anything about layout.